client/src/locales/en.json

```diff
@@ -216,6 +216,11 @@
   "dashboard.index.no-arch-sacks": "There are no archived sacks yet.",
   "dashboard.index.no-sacks": "There are no sacks yet.",
   "dashboard.index.show-arch-sacks": "Show archived sacks",
+  "delete-link.has-expired-part1": "Delete link has expired. Please go to",
+  "delete-link.has-expired-link": "user profile page ",
+  "delete-link.has-expired-part2": "and find delete account option to get new delete link.",
```

Comment:
Do not cut sentences in the middle - use parameter if needed. It is hard or impossible to translate such divided sentences.